app/services/finance/participants/update.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Finance
+  module Participants
+    class Update
+      def initialize(participant_profile)
+        self.participant_profile = participant_profile
+      end
+
+      def call(attributes)
```

Comment:
This probably appears for now overkill. We have however more fields to update in the pipeline, and this may become more relevant. If it remains overkill i.e `participant_profile.update!(attributes)` is the only thing we need to do then we can get rid of this. 
But if someone feels strongly about it I'm happy to remove it and have the controller perform the update.